plugins/romloader/jtag/openocd/tcl/jtag_detect_init.tcl

```diff
@@ -592,9 +592,10 @@ proc netX90_COM_disable_irqs {} {
 	bp 0x0006000E 2 hw
 	resume
 	echo "CPSID executed"
+	set HALT_TIMEOUT 2000
 	if {[catch {halt $HALT_TIMEOUT} err] == 0} {
```

Comment:
It should be wait_halt. We don't want to halt the CPU, it should run up to the breakpoint and halt by itself. 
(Even though it may not matter because I expect that the netx has already halted by the time openOCD processes the halt command).